install/update_94_95.php

```diff
@@ -1513,60 +1513,6 @@ function update94to95() {
    $migration->addConfig(['ssologout_url' => '']);
    /** SSO logout URL */
 
-   /** A doc_item to rule them all! */
-   $migration->dropKey('glpi_documents_items', 'unicity');
-   $migration->migrationOneTable('glpi_documents_items');
-   $migration->addKey(
-      'glpi_documents_items',
-      ['documents_id', 'itemtype', 'items_id', 'timeline_position'],
-      'unicity',
-      'UNIQUE'
-   );
-   $migration->migrationOneTable('glpi_documents_items');
```

Comment:
This should stay here. DB already migrated on GLPI 9.5.x have correct `unicity` key, no need to rebuilt it for them.